app/src/androidTest/java/com/github/se/project/ui/endToEnd/TutorEndToEnd.kt

```diff
@@ -217,4 +262,5 @@ class TutorEndToEndTest {
     composeTestRule.onNodeWithTag("lessonCard_0").assertExists()
     composeTestRule.onNodeWithText("Student: Ozymandias Halifax").assertIsDisplayed()
   }
-}*/
+}
+ */
```

Comment:
The changes from this files are mine since you went from my branch right?

Reply:
These are coming from the automated merge of the main indeed